formwidgets/sitemapitems/partials/_item.htm

```diff
@@ -10,7 +10,7 @@
             <span class="title"><?= e($this->getReferenceDescription($item)) ?></span>
             <span class="comment">
                 <?= e(trans('winter.sitemap::lang.item.priority')) ?>: <span class="priority"><?= e($item->priority) ?></span>,
-                <?= e(trans('winter.sitemap::lang.item.changefreq')) ?>: <span class="changefreq"><?= e($item->changefreq) ?></span>
+                <?= e(trans('winter.sitemap::lang.item.changefreq')) ?>: <span class="changefreq"><?= e(trans('winter.sitemap::lang.item.' . $item->changefreq)) ?></span>
```

Comment:
What's the intention of this change?

Reply:
The values of frequency select box are translated. When I add a new item, it shown translated. But I refresh the page, the frequency is not translated.